app/renderer/settings.js

```diff
@@ -187,6 +189,7 @@ const defaultSettings = {
     "splitright": false,
     "startuppages": "",
     "storenewvisits": "pages",
+    "suggestbookmarks": "all",
```

Comment:
Should be part of `suggestorder`, please see my later comments in #100, or let me know if you have any questions on this. Ideally there should "only" be a `bookmarksfile` setting to and the file contains all the bookmarks, and how they are suggested is controlled by `suggestorder` like other suggestions.

Reply:
I think this is what you mentioned in #100 that the setting `suggestbookmarks` was going to refer to which bookmarks should be included in the suggestions. Maybe we could use this to specify only bookmarks containing a certain tag to be suggested, and others only be accessible through `:bmload`. 
Other option could be to add it as a flag on the bookmark itself.

Reply:
Sorry for that, yes you are right. `suggestorder` is only for if/where/how many suggestions should show, `suggestbookmarks` is for optionally filtering which tags should be listed (or default to all). My comment about `suggestbookmarks` not affecting anything that happens in the bookmarksfile still stands, and as such you don't need to write anything to the file after changing the value.

Reply:
Can you remove "all" from here and just make it a list of tags to filter? And if empty, just show all?

Reply:
Done!